draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
+        |                    |                           |              .
+  time(EG)                   |                           |   {}-on-wire .
+        |----Evidence------->|                           |              .
+        |    {time(EG)}   time(ER)--Evidence{time(EG)}-->|              .
+        |                    |                        time(RG)          .
+        |                 time(RA)<-Attestation Result---|              .
+        |                    |        {time(RX)}         |              .
+        ~                    ~                           ~              .
+        |                 time(HD')<------------------------------------/
+   time(HD')<-----------------------------------------------------------/
+        |                    |                        time(HD')<--------/
         |                 time(OP)                       |
 ~~~~
 
-In comparison with example 1, the time considerations in this example
-go into more detail with respect to the life-cycle of Claims and
-Evidence. While the goal is to create up-to-date and recent Evidence as soon as possible, typically there is a latency between value generation and Attester awareness.
-
-At time(AA) the Attesting Environment is able to trigger an event
-(e.g. based on an Event-Condition-Action model) to create attestation
-Evidence that is as recent as possible. In essence, at time(AA) the
-Attesting Environment is aware of new values that where generated at
-time(VG) and corresponding Claim values are collected immediately.
-Consecutively, Evidence based on relevant "old" Claims and the just
-collected "new" Claims is generated at time(EG). In essence, the Claims used to generate the Evidence are generated at various time(VG) before time(AA).
-
-In order to create attestation Evidence at
-at time(AA), the Attester requires a fresh (i.e. not expired)
-centrally generated handle that has been distributed to all involved
-entities.
-
-In general, The duration a handle remains fresh depends on
-the content-type of the handle. If it is a (relative or absolute)
-timestamp, clocks synchronized with a shared and trustworthy source of
-time are required. If another value type is used as a handle, the
-reception time of the handle time(HD) provides an epoch (relative time
-of zero) for measuring the duration of validity (similar to a
-heart-beat timeout). From the point of view of a Verifier, validity of
-Evidence is only given if the handle used in Evidence satisfies
-delta(time(HD),time(EG))distribution-interval.
-
-In this usage scenario, time(VG), time(AA), and time(EG) are tightly
-coupled. Also, the absolute point in time at which a handle is received
-by all three entities is assumed to be close to identical.
-
-## Example 4: Nonce-based Background-Check Model Example
+At time(AA), the Attesting Environment is able to trigger an event (e.g. based on an Event-Condition-Action model) to generate Evidence as soon/recent as possible at time (EG).
```

Comment:
Why do you need time(AA) and time(EG) both, instead of just using one, like time(EG) as in other examples?  In particular, there is no security check mentioned below that uses time(AA) and so that event should not be added

Reply:
Because time(AA) does not seem to be relevant wrt how solutions work in the other examples. Yet it is the vital point in time that drives this example (potentially no clocks involved, a lot of types of handles allowed). We wrote text version multiple times until now and it results in a strange disconnect when you are very explicit about time(foo) and then suddenly you just use English text to express: "and then the most important time is when the Attesting Environment becomes aware of changes to some relevant values in Target Environments, starts to collect them in order to create the semantic Claims and finally generates the Evidence at time(EG)". There is a very confusing disconnect for the reader, why only the last time is expressed so explicitly, while the potential latency between VG and AA is the driver (or disabler!) for secure evidence here.

We are now maybe too reluctant to add more complexity and text right now. Maybe that is the primary cause for confusion, still. The security check you expect here is other an implementation specific one (is that okay to highlight here in the architecture) or not done by the Attester but the Verifier or even at the Attestation Result appraisal stage at the Relying Party. That would create a lot of text to spell all that out. Maybe there is a simpler way to phrase it? When we read this in the editor's session, we should discuss this live.